src/CallToAction.elm

```diff
@@ -18,8 +18,11 @@ callToActionButton callToAction aClass =
                 [ getIcon callToAction.icon (Just "button--icon")
                 , span [] [ text callToAction.promptLong ]
                 ]
-            , span []
-                [ text callToAction.displayHref ]
+                ,
```

Comment:
I'm not sure about the formatting. If you have `elm-format` installed can you try running to see if it changes?

Reply:
yes nick just told me about it. i will install now.